x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_creation/create_new_terms.ts

```diff
@@ -9,10 +9,10 @@ import expect from '@kbn/expect';
 
 import { DETECTION_ENGINE_RULES_URL } from '@kbn/security-solution-plugin/common/constants';
 import { getCreateNewTermsRulesSchemaMock } from '@kbn/security-solution-plugin/common/api/detection_engine/model/rule_schema/mocks';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+
 import { deleteAllRules } from '../../utils';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
 
-// eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext) => {
```

Comment:
Do we need to add the tags to the test too?

Reply:
🙈

Reply:
Done https://github.com/elastic/kibana/pull/170025/commits/a9d1ffaf81bf54cdc3c292943688f2ff9ce51f80